parts/page.html

```diff
@@ -0,0 +1,5 @@
+<!-- wp:group {"tagName":"article","layout":{"inherit":true}} -->
+<article class="wp-block-group"><!-- wp:post-title {"level":1} /-->
+
+	<!-- wp:post-content /--></article>
```

Comment:
@tropicandid seeing this alignment/structure as being "off" in a number of places and so maybe this is how WordPress is generating these files?